demo/src/components/MiniCart/MiniCart.js

```diff
@@ -0,0 +1,60 @@
+import {Container, Col, Row} from 'react-bootstrap';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {computePrice} from '../Cart/Cart.js';
+import '../Cart/Cart.css';
+
+/**
+ * Creates a component describing a shopping cart.
+ * @param {!Object<string,
+ *      {name:string, item:!Object, quantity:number, description:string,
+ *      inCart:boolean, cost:number}>} items The items stored in the site state.
+ * @param {function(string, number)} setQuantity A function to modify
```

Comment:
It looks like this method does not have a setQuantity param.

Reply:
Removed it.